src/clients/BundleDataClient.ts

```diff
@@ -216,20 +241,37 @@ export class BundleDataClient {
       hubPoolClient.chainId
     );
     const chainIds = this.clients.configStoreClient.getChainIdIndicesForBlock(nextBundleMainnetStartBlock);
-    const futureBundleEvaluationBlockRanges = getWidestPossibleExpectedBlockRange(
-      chainIds,
-      this.spokePoolClients,
-      getEndBlockBuffers(chainIds, this.blockRangeEndBlockBuffer),
-      this.clients,
-      this.clients.hubPoolClient.latestBlockSearched,
-      this.clients.configStoreClient.getEnabledChains(this.clients.hubPoolClient.latestBlockSearched)
-    );
+
+    // We need to find the widest possible range of blocks to search for refunds. Since we aim to leverage
+    // Arweave's DA layer to look for entries, we need to make sure we're using an expected range. If we
+    // have a pending proposal, we can assume that a corresponding Arweave entry exists. In this case, we
+    // need to capture the `impliedBlockRange` so that it matches with the tag in the Arweave entry. If no
+    // pending bundle exists, we should cast as large of a next bundle range as possible to capture the most
+    // data. We can do this with `getWidestPossibleExpectedBlockRange` which is a superset of the ranges for
+    // `impliedBlockRange`.
```

Comment:
Per comments above ^^

```suggestion
    // data. We can do this with `getWidestPossibleExpectedBlockRange`.
```